README.md

```diff
@@ -2,53 +2,92 @@
 
 ## Introduction
 
-CWL implementations of simtools workflows:
+CWL implementations of workflows for the CTA Simulation Pipeline.
 
-- workflows for setting and validation of parameters are in [./workflows](./workflows).
-- command line tools used in these workflows and calling simtools applications are in [./workflows/tools](./workflows/tools).
-- input and generated data is validated using schemas located in [./schemas](./schemas).
+Workflows are used to set, derive, and validate simulation model parameters following all or a subset of these steps:
 
-## Getting started
+Setting Workflows:
 
-Clone workflows:
-```
-git clone git@github.com:gammasim/workflows.git
-```
+1. Receive: data or parameter update through an API.
+2. Assert: verify that input data or parameters are in correct units, formats, allowed ranges, etc.
+3. Derive: derive simulation model parameter from received data (not applicable to all input data).
+4. Validate: validate updated simulation model parameter (e.g., by comparison with measurements).
+5. Review: review of updates to simulation model parameter(s) and validation steps (by expert; in some cases automatized).
 
-Install dependencies using mamba or conda:
-```
+Acceptance Workflows:
+
+1. Feedback: provide feedback on validation and review (successful / unsuccessful validation).
+2. Update DB: update simulation model database with new parameter (successful validation).
+3. Document: document all derivation and validation steps.
+
+The implementation in *simtools-workflows* consist of the following main components:
+
+- **workflows** encoded in [Common Workflow Language (CWL)](https://www.commonwl.org/) (see [./workflows](./workflows) in this repository)
+- **parameter schemas** defining all input data and simulation model parameters (see [./schemas](./schemas) in this repository)
+- **simulation tools and software** consisting of [simtools](https://github.com/gammasim/simtools) and the simulation software (e.g., [CORSIKA](https://www.iap.kit.edu/corsika/) and [sim_telarray](https://www.mpi-hd.mpg.de/hfm/~bernlohr/sim_telarray/)).
```

Comment:
Good point, I've changed the sentence so that it is clearer that simtools are not part of the workflows. For containers: I seem them now as part of simtools, so not really necessary to mention them again.